addOns/graphql/src/main/resources/org/zaproxy/addon/graphql/resources/Messages.properties

```diff
@@ -5,6 +5,7 @@ graphql.api.action.importFile.param.file = The File That Contains the GraphQL Sc
 graphql.api.action.importFile.param.endurl = The Endpoint URL.
 graphql.api.action.importUrl = Retrieves a GraphQL Schema from a URL.
```

Comment:
`Imports ...` to be consistent with the file description, same for the command line.